### DISCUSSION TITLE
New ideas?

### DISCUSSION THREAD

**Discussion by chrislemke** [chrislemke/ChatFred#29] [Ideas]
Hey! 
I just released a [new version](https://github.com/chrislemke/ChatFred/releases/tag/v1.3.0). With some new features. 

But I am still looking for new ideas. Please feel free to share them here! Thanks!

**Comment by mindtheflo** [comment]
Hi @chrislemke , amazing workflow! 
I added a Shift modifier key to the CF command and fallback search to speak the answer. I like to be able to tell the workflow when I want to hear the answer v.s read it.
An other thing that would be nice is a keyword to reset the context and erase the .csv history. 
Thanks again!

**Reply by chrislemke** [reply]
Hey @parisetflorian!
Cool ideas. If you want you can create a PR and we merge your changes to the project. If you want to delete the .csv file you can actually just type: `cf forget me` or any other command in [`clear_log_prompts`](https://github.com/chrislemke/ChatFred/blob/4bdab51d91b16d17ec5c7c2707f9bb22967fc6e0/workflow/src/custom_prompts.py#L19).
